### DISCUSSION TITLE
"You are not authenticated!" when trying to login

### DISCUSSION THREAD

**Discussion by MCKDiana** [UiPath/orchestrator-powershell#163] [Q&A]
Hello. I am getting "You are not authenticated!" error message when I try to run 

Get-UiPathAuthToken -Session -URL <orchestrator_url> -Username <username> -Password <password>. 

I have an on-premise Orchestrator. 

I am just trying to Login to UiPath. Is there something else I need to do or try?

**Comment by codemonkee** [comment]
Late to the part on this question... Try downgrading the module you are using. I had similar challenge that I cover in the other Question on here.

- What version of Orchestrator?
- What version(s) of the `UiPath.PowerShell` Module have you tried?

```
Install-Module -Name 'UiPath.PowerShell' -MaximumVersion '20.4.2.13' -Force
Import-Module UiPath.PowerShell -MaximumVersion 20.4.2.13
$OrchestratorUrl = 'url.domain'
$OrchestratorHostTenant = 'host'
$OrchestratorDefaultTenant = 'default'

#local account
$UiPathAuthParams = @{
  Session = $True
  URL = $OrchestratorUrl
  TenantName = $OrchestratorDefaultTenant
  Username = 'username'
  Password = (Read-Host "Enter Orchestrator Password" -AsSecureString)
}
Get-UiPathAuthToken @UiPathAuthParams -Password ($UiPathAuthParams.Password | ConvertFrom-SecureString -AsPlainText)

##Authenticating using Windows Credentials would look something like
Get-UiPathAuthToken -WindowsCredentials -Session -URL $OrchestratorUrl -FolderPath $OrchestratorDefaultTenant
```
